lib/mutant/cli/command/environment.rb

```diff
@@ -103,10 +102,7 @@ def add_matcher_options(parser)
             add_matcher(:start_expressions, @config.expression_parser.call(pattern).from_right)
           end
           parser.on('--since REVISION', 'Only select subjects touched since REVISION') do |revision|
-            add_matcher(
-              :subject_filters,
```

Comment:
@dgollahon I could get a rid of the entire `subject_filters` after checking private integrations, they where ported to the hooks subsystem so no need to maintain this API. this overall makes this commit much nicer than previous versions.

Reply:
Awesome! Thank you for validating and simplifying that. I think it's much nicer code-wise, just didn't realize it was an option.